.env.example

```
@@ -1,3 +1,7 @@
+# Sentry error monitoring
+NUXT_PUBLIC_SENTRY_DSN=
+SENTRY_AUTH_TOKEN= # build-time only (sourcemap upload); do not set at runtime
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)